documents/toolbox-population.md

```diff
@@ -0,0 +1,80 @@
+# Toolbox population
+
+## Toolbox items from unreferenced assemblies (all supported versions of Visual Studio)
+
+This approach requires registering with the Toolbox Controls Installer (TCI) in the Windows registry. [The TCI spec](https://www.microsoft.com/en-us/download/details.aspx?id=35536) explains how to register your assemblies. Once registered, your controls will be shown in the Toolbox for any compatible project, even projects that do not reference your assembly.
+
+### Limitations
+
+* This approach only works reliably for WPF .NET Framework assemblies. The controls in registered assemblies are enumerated via .NET Reflection which does not work reliably on .NET Core assemblies within a .NET Framework process such as Visual Studio.
```

Comment:
This makes it sound like it may work for .NET Core but not reliably. We should be explicitly stating that this is only supported for .NET Framework and not .NET Core.

Reply:
Fixed